rules/okta_rules/okta_org2org_creation_modification.py

```diff
@@ -0,0 +1,39 @@
+from panther_base_helpers import get_val_from_list, okta_alert_context
+
+APP_LIFECYCLE_EVENTS = (
+    "application.lifecycle.update",
+    "application.lifecycle.create",
+    "application.lifecycle.activate",
+)
+
+
+def rule(event):
+    if event.get("eventType") not in APP_LIFECYCLE_EVENTS:
```

Comment:
Same comment re: default.

Reply:
Default value of None is ok in boolean logic imo